pkg/machine/env/dir.go

```diff
@@ -164,9 +165,13 @@ func GetSSHIdentityPath(name string) (string, error) {
 	return filepath.Join(datadir, name), nil
 }
 
-func WithPodmanPrefix(name string) string {
-	if !strings.HasPrefix(name, "podman") {
-		name = "podman-" + name
+func WithToolPrefix(name string) string {
+	toolName := os.Getenv("PODMAN_TOOL_PREFIX")
+	if toolName == "" {
+		toolName = "podman"
+	}
```

Comment:
Might be worth wrapping this in https://pkg.go.dev/sync#OnceValue to avoid calling `Getenv` everytime.

Reply:
Done 👍